src/styles/_catalog.scss

```diff
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
+    cursor: pointer;
+    transition-duration: color 300ms, transform 300ms;
+    transition-timing-function: linear;
```

Comment:
woah, too much transitions :)
 `transition` is a shorthand property, if you specify it, you don't need any else
 more info: https://developer.mozilla.org/en-US/docs/Web/CSS/transition